supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -179,11 +179,12 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
                                            (implicit gcsTransferConfiguration: GcsTransferConfiguration): String = {
     // Generate a mapping of reference inputs to their mounted paths and a section of the localization script to
     // "faux localize" these reference inputs with symlinks to their locations on mounted reference disks.
+    import cromwell.backend.google.pipelines.common.action.ActionCommands._
     val referenceFilesLocalizationScript = {
       val symlinkCreationCommandsOpt = referenceInputsToMountedPathsOpt map { referenceInputsToMountedPaths =>
         referenceInputsToMountedPaths map {
           case (input, absolutePathOnRefDisk) =>
-            s"mkdir -p ${input.containerPath.parent.pathAsString} && ln -s $absolutePathOnRefDisk ${input.containerPath.pathAsString}"
+            s"mkdir -p ${input.containerPath.parent.pathAsString.escape} && ln -s ${absolutePathOnRefDisk.escape} ${input.containerPath.pathAsString.escape}"
```

Comment:
Are there other places we might be vulnerable to this, or is there something specific about the way reference disk paths are handled that bypasses other escaping?

Reply:
I believe Cromwell is appropriately robust to these potential filename issues in the "normal" localization code paths thanks to a liberal use of [this shellEscaped function](https://github.com/broadinstitute/cromwell/blob/f14d91a3845622bef0bebc7c1aba41d9aaf91aa6/supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala#L43). We were able to localize this particular file just fine (apart from the slowness) when we weren't using the reference disk.

Reply:
Ah, nice! Is there a reason we can't use that function here?

Reply:
We could definitely use that function here if that's the preference. I modeled these changes on the existing `implicit class` [ShellPath](https://github.com/broadinstitute/cromwell/blob/7bcb7b4b302a29264b2a97d833efa5b5eb0015b5/supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionCommands.scala#L31-L34) which allows for invoking `.escape` as if it were a method on `String`. If using `shellEscaped` this would (arguably) look a little noisier:

```suggestion
            s"mkdir -p ${shellEscaped(input.containerPath.parent.pathAsString)} && ln -s ${shellEscape(absolutePathOnRefDisk)} ${shellEscape(input.containerPath.pathAsString)}"
``````

Reply:
I prefer minimizing the number of paths for string escaping... but also acknowledge that this is a total nitpick. :)